.eslintrc.js

```diff
@@ -1,9 +1,18 @@
 const config = {
     parser: 'babel-eslint',
+    plugins: ['react'],
     extends: ['@magento'],
     rules: {
         'no-undef': 'off',
-        'no-useless-escape': 'off'
+        'no-useless-escape': 'off',
+        'react/jsx-no-literals': [
+            'error',
+            {
+                allowedStrings: [],
+                ignoreProps: true,
+                noStrings: true
+            }
+        ]
```

Comment:
I decided to add the rule to our eslint file as I don't want to introduce an error rule to the generic `magento-eslint`.

I'm not against it, but I think it's a bit opinionated and only applies to us because we are using i18n/format message and want to use this rule to not forget strings.

Reply:
The props that we would like it to pick up are mostly for `label` and `title`. Doesn't seem to be a way to allow those, sadly.